upload_recipe_form.html

```diff
@@ -7,118 +7,103 @@
-			<input type="text" name="time" id="time" placeholder="type recipe time here">
-			<br>
-			<label for=yield>Yield:</label>
-			<input type="text" name="yield" id="yield" placeholder="type recipe yield here">
-			<br>
-		</fieldset>
-		<fieldset name="recipe_details">
-			<legend>Recipe Details</legend>
-			<label for="equipment">Equipment:</label>
-			<input type="text" name="equipment" id="equipment"  placeholder="type equipment here">
-			<input type="button" name="add_equipment" value="Add Equipment" onclick="addEquipment()">
-			<br>
-			<label for="equipment_list">List of Equipment</label>
-			<ul id="equipment_list"></ul>
-			<br>
-			<label for=ingredients>Ingredients (with quantities):</label>
-			<br>
-			<input type="text" name="ingredients" id="ingredient" placeholder="type ingredient here">
-			<input type="button" name="add_ingredient" value="Add Ingredient" onclick="addIngredient()">
-			<br>
-			<label for="ingedient_list">List of Ingredients</label>
-			<ul id="ingredient_list"></ul>
-			<br>
-			<label for="instruction">Procedure:</label>
-			<br>
-			<textarea name="instruction" id="instruction" rows="10" cols="50" placeholder="Type instruction here"></textarea>
-			<br>
-			<input type="button" name="add_instruction"
-			id="add_instruction" value="Add Instruction"
-			onclick="addInstruction()">
-			<br>
-			<label for="instruction_list">List of Instructions</label>
-			<ol id="instruction_list"></ol>
-		</fieldset>
-		<fieldset name="add_tags_to_recipe">
-			<legend>Add Tags to the Recipe</legend>
-			<br>
-			<label for="tag_checkboxes">Select Your Tags:</label>
-			<div name="tag_checkboxes" id="tag_checkboxes">
-			<label for=tag1>Breakfast</label>
-			<input type="checkbox" name="tag" id="tag1" value="Breakfast">
-			<br>
-			<label for=tag2>Lunch</label>
-			<input type="checkbox" name="tag" id="tag2" value="Lunch">
-			<br>
-			<label for=tag3>Dinner</label>
-			<input type="checkbox" name="tag" id="tag3" value="Dinner">
-			<br>
-			<label for=tag4>Dessert</label>
-			<input type="checkbox" name="tag" id="tag4" value="Dessert">
-			<br>
+			<div name="medialink" id="medialink">
+				<label for="medialink">Optional: Provide Supplemental Audio or Video to the Recipe</label>
+				<br>
+				<input type="radio" name="mediatype" id="audio" value="Audio">
+				<label for="audio">Audio</label>
+				<br>
+				<input type="radio" name="mediatype" id="video" value="Video">
+				<label for="video">Video</label>
+				<br>
+				<input type="radio" name="mediatype" id="nomedia" value="No Supplemental Media">
+				<label for="nomedia">No Supplemental Media</label>
+				<br>
+				<label for="media_url">Link:</label>
+				<input type="url" name="media_url" id="media_url" placeholder="Link to audio/video here">
+			</div>
+			<div name="recipeinfo" id="recipeinfo">
+				<br>
+				<label for="recipeinfo">Information About the Recipe</label>
+				<br>
+				<label for="recipe_name">Recipe Name: </label>
```

Comment:
this is only a nitpick: can we ask for recipe name and then the supplemental media? Asking for name first is kind of an orientation technique akin to titling a document. It's a signpost that orients users as to what they're doing.

Reply:
Resolved